webpack.config.js

```diff
@@ -75,9 +76,12 @@ module.exports = {
       NODE_ENV: JSON.stringify(process.env.NODE_ENV),
     }),
     new ForkTsCheckerWebpackPlugin(),
-    new CopyWebpackPlugin([
-      { from: 'public' },
-    ]),
+    new CopyPlugin({
+      patterns: [
+        { from: 'public', to: path.resolve(__dirname, 'public') },
+      ],
+    }),
+    new NodePolyfillPlugin(),
```

Comment:
These two changes are a result of using webpack 5. `CopyWebpackPlugin` was causing a build error due to `from` being an invalid options property. Webpack 5 no longer provides polyfills, and using `NodePolyfillPlugin` resolves that.